stdlib/universe/universe.flux

```diff
@@ -237,6 +237,80 @@ builtin string : (v: A) => string
 builtin time : (v: A) => time
 builtin uint : (v: A) => uint
 
+// display converts any value into a string representation.
+// This function differs from the `string` function in that it will convert any value
+// to a string recusively for all composite types.
+//
+//  Basic types are converted to a string directly.
+//  Bytes are represented as a string of lowercase hexadecimal characters prefixed with `0x`.
+//  Composite types (i.e. arrays, dictionaries, records) will be represented in a syntax similar
+//  to the equivalent Flux literal representation.
+//
+//  It is not expected that the representation can be parsed back into the original value.
+//  The representation may span multiple lines.
+//  The representation is not stable across Flux versions.
+//
+//  ## Parameters
+//  - v: Value to be converted for display.
+//
+//  ## Examples
+//
+//  ### Display a record
+//
+//  ```no_run
+//  x = {a: 1, b: 2, c: 3}
+//  display(v: x)
+//
+//  // Returns {a: 1, b: 2, c: 3}
+//  ```
+//
+//  ### Display an array
+//
+//  ```no_run
+//  x = [1, 2, 3]
+//  display(v: x)
+//
+//  // Returns [1, 2, 3]
+//  ```
+//
+//  ### Display a dictionary
+//
+//  ```no_run
+//  x = ["a": 1, "b": 2, "c": 3]
+//  display(v: x)
+//
+//  // Returns [a: 1, b: 2, c: 3]
+//  ```
+//  ### Display bytes
+//
+//  ```no_run
+//  x = bytes(v:"abc")
+//  display(v: x)
+//
+//  // Returns 0x616263
+//  ```
+//
+//  ### Display a composite value
+//
+//  ```no_run
+//  x = {
+//      bytes: bytes(v: "abc"),
+//      string: "str",
+//      array: [1,2,3],
+//      dict: ["a": 1, "b": 2, "c": 3],
+//  }
+//  display(v: x)
+//
+//  // Returns
+//  // {
+//  //    array: [1, 2, 3],
+//  //    bytes: 0x616263,
+//  //    dict: [a: 1, b: 2, c: 3],
+//  //    string: str
+//  // }
+//  ```
+builtin display : (v: A) => string
```

Comment:
Should we use the name format instead?

Reply:
The name `format` implies some kind of control over how the value is represented and that is explicitly not the intent of this function. This function is opinionated and can change but it always gives a string representation that is human readable.